src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/MS/Internal/Tasks/TaskHelper.cs

```diff
@@ -52,13 +52,10 @@ internal static class TaskHelper
         // </summary>
         internal static void DisplayLogo(TaskLoggingHelper log, string taskName)
         {
-            string acPath = Assembly.GetExecutingAssembly().Location;
-            FileVersionInfo acFileVersionInfo = FileVersionInfo.GetVersionInfo(acPath);
-
-            string avalonFileVersion = acFileVersionInfo.FileVersion;
+            Version avalonFileVersion = Assembly.GetExecutingAssembly().GetName().Version;
```

Comment:
Isn't this the assemby version, not file version? I think we could get the file version of the executing process if that's more appropriate. It probably depends on whether we're using the apphost